docs/docs/migration-guide.mdx

```diff
@@ -33,13 +33,14 @@ lock_store:
     key_prefix: <alphanumeric value to prepend to lock store keys>
 ```
 
-Note that not specifying the `concurrent_mode` parameter or setting it to `false` will default to using the `RedisLockStore` implementation.
+Note that omitting the `concurrent_mode` parameter or setting it to `false` will default to using the `RedisLockStore`.
 
-No database migration is required when switching to the `ConcurrentRedisLockStore`, and you may use the same Redis database number.
+No database migration is required when switching to the `ConcurrentRedisLockStore`. You can use the same Redis instance and database number as you did previously when using the `RedisLockStore`.
 You may want to delete all the preexisting keys if using the same Redis database number.
-The reason for this is because what used to be stored by the `RedisLockStore`(i.e., serialized [TicketLock](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticketlock-objects) instances) is no longer stored by the new `ConcurrentRedisLockStore`.
-The `ConcurrentRedisLockStore` instead stores individual `Ticket` instances, as well as the last issued ticket number.
-The mechanism for concurrent message handling of the same conversation id recreates the `TicketLock` from the persisted `Ticket` instances.
+These former key-value items are no longer required by the `ConcurrentRedisLockStore` and the database can be cleared.
+
+There is no overlap in key-value items stored when using the `RedisLockStore` and the `ConcurrentRedisLockStore`, because the `RedisLockStore` persists serialized [TicketLock](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticketlock-objects) instances) while the `ConcurrentRedisLockStore` instead stores individual [`Ticket`](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticket-objects) instances, as well as the last issued ticket number.
```

Comment:
```suggestion
There is no overlap in key-value items stored when using the `RedisLockStore` and the `ConcurrentRedisLockStore`, because the `RedisLockStore` persists serialized [TicketLock](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticketlock-objects) instances while the `ConcurrentRedisLockStore` instead stores individual [`Ticket`](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticket-objects) instances, as well as the last issued ticket number.
```